api/src/Security/KeycloakAccessTokenHandler.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Security;
+
+use App\Exception\InvalidUserException;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Security\Core\Exception\BadCredentialsException;
+use Symfony\Component\Security\Http\AccessToken\AccessTokenHandlerInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * Send token to Keycloak to validate it.
+ *
+ * @author Vincent Chalamon <vincentchalamon@gmail.com>
+ *
+ * @final
+ */
+class KeycloakAccessTokenHandler implements AccessTokenHandlerInterface
```

Comment:
```suggestion
class OidcAccessTokenHandler implements AccessTokenHandlerInterface
```

As this should work with any auth server implementing OIDC.